server/sonar-db-dao/src/main/resources/org/sonar/db/issue/IssueMapper.xml

```diff
@@ -213,6 +213,18 @@
     </foreach>
   </select>
 
+  <select id="selectResolvedOrConfirmedByComponentUuid" parameterType="String" resultType="Issue">
+    select
+    <include refid="issueColumns"/>
+    from issues i
+      inner join rules r on r.id=i.rule_id
+      inner join projects p on p.uuid=i.component_uuid
+      inner join projects root on root.uuid=i.project_uuid
```

Comment:
We probably don't need all this, so perhaps we can create a lighter project and get rid of all the joins

Reply:
Yeah, this was lazy of me to try to reuse the existing mapper. I'll create a new light-weight object to map to, with only the fields we need. Should perform better.